tests/test_ai_disclosure.py

```
@@ -0,0 +1,165 @@
+    assert text.rstrip().endswith(AI_DISCLOSURE_TEXT), text[-200:]
+    assert "<img" not in text
+    assert AI_DISCLOSURE_ICON_URL not in text
+    assert "http" not in AI_DISCLOSURE_TEXT  # die Zeile selbst ist link-frei
+    print("  linkedin_ends_with_disclosure_text_only OK")
+
+
+# --- 2 · Markdown: Einzelanalyse UND Modellvergleich ----------------------
+
+def _assert_markdown_footer(doc: str, label: str) -> None:
+    assert AI_DISCLOSURE_MARKDOWN in doc, f"{label}: Fußblock fehlt"
+    assert f'width="90"' in doc, f"{label}: img width fehlt"
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)